examples/light-switch-app/asr/src/BindingHandler.cpp

```diff
@@ -280,7 +280,7 @@ void BindingHandler::PrintBindingTable()
             \t+ ClusterId %d \n \
             \t+ RemoteEndpointId %d \n \
             \t+ NodeId %d",
-                    (int) entry.fabricIndex, (int) entry.local, (int) entry.clusterId.Value(), (int) entry.remote,
+                    (int) entry.fabricIndex, (int) entry.local, (int) entry.clusterId.value_or(-1), (int) entry.remote,
```

Comment:
```suggestion
                    (int) entry.fabricIndex, (int) entry.local, (int) entry.clusterId.value_or(kInvalidClusterId), (int) entry.remote,
```
because `-1` is not even the right type for a `ClusterId`.